### PR TITLE
arc_monitoring_servers docs

### PR DESCRIPTION
**Pull request by alsanch**
arc_monitoring_servers docs

### CODE REVIEW

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 1]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
   - AZURE MONITOR
----
\ No newline at end of file
+---
+
+# Infrastructure observability for Azure Arc-enabled servers using Azure Monitor
+
+## Overview
+
+Infrastructure observability is crucial for Contoso Motors to understand the performance and health of their Azure Arc-enabled servers. This is where [Azure Monitor](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/hybrid/arc-enabled-servers/eslz-management-and-monitoring-arc-server) comes into play, playing a vital role in offering visibility into every facet of their Azure Arc-enabled servers ecosystem.
```
Comment (janegilring) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:18]:
```suggestion
Infrastructure observability is crucial for Contoso Motors to understand the performance and health of their Azure Arc-enabled servers. This is where [Azure Monitor](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/hybrid/arc-enabled-servers/eslz-management-and-monitoring-arc-server) steps in, playing a vital role in offering visibility into every facet of their Azure Arc-enabled servers ecosystem.
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 2]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+- You will find the DCR that has been created to collect insights from the Azure Arc-enabled servers:
+
+    ![Screenshot of the Data Collection Rules](./img/dcr_vmi.png)
+
+- Click on the DCR. You will see the data sources collected, in this case, performance counters:
+
+    ![Screenshot of the DCR - Data sources](./img/dcr_datasources.png)
+
+## Azure Arc-enabled servers and Azure Monitor VMInsights Integration
```
Comment (janegilring) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:50]: We should use the same casing as https://learn.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-overview

```suggestion
## Azure Arc-enabled servers and Azure Monitor VM insights Integration
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 3]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+Now that we have checked that the required monitoring artifacts have been successfully enabled, it's time to leverage VMInsights. It monitors the performance of your Azure Arc-enabled servers by collecting the required data with AMA.
```
Comment (janegilring) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:52]: We should use the same casing as https://learn.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-overview
```suggestion
Now that we have checked that the required monitoring artifacts have been successfully enabled, it's time to leverage VM insights. It monitors the performance of your Azure Arc-enabled servers by collecting the required data with AMA.
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 4]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+- Search for **Azure Arc**, go to **Machines** and click in one of your **Azure Arc-enabled servers**:
+
+    ![Screenshot of searching for an Azure Arc-enabled server](./img/search_arc_server.png)
+
+- Click on **Insights** and then on **Performance**. You will find a set of performance charts that target several key performance indicators to help you determine how well your Azure Arc-enabled server is performing. The charts show resource utilization over a period of time:
+
+    ![Screenshot of VMInsights - Performance](./img/vminsights_performance.png)
```
Comment (janegilring) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:60]: We should use the same casing as https://learn.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-overview
```suggestion
    ![Screenshot of VM insights - Performance](./img/vminsights_performance.png)
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 5]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+## Operating System (OS) Performance Workbook
+
+An Azure Monitor Workbook for Operating System (OS) Performance is also available as part of the deployment. It complements the views provided by VMInsights.
```
Comment (janegilring) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:64]: We should use the same casing as https://learn.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-overview
```suggestion
An Azure Monitor Workbook for Operating System (OS) Performance is also available as part of the deployment. It complements the views provided by VM insights.
```

**Review by janegilring — APPROVED** [review]
Looks very good, added some minor suggestions on wording and casing.

**Comment by fcabrera23** [comment]
@alsanch could you please rebase this branch based on the latest canary branch? I've added the configurations to get the Vale linitn check

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 6]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+## Overview
+
+Infrastructure observability is crucial for Contoso Motors to understand the performance and health of their Azure Arc-enabled servers. This is where [Azure Monitor](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/hybrid/arc-enabled-servers/eslz-management-and-monitoring-arc-server) steps in, playing a vital role in offering visibility into every facet of their Azure Arc-enabled servers ecosystem.
+
+Azure Monitor empowers Contoso with the capability to monitor and gather telemetry data from their Azure Arc-enabled servers. It serves as a central hub, providing near real-time insights into server performance, health, and resource utilization. Azure Monitor offers a comprehensive view of the entire infrastructure, ensuring proactive identification and resolution of potential issues.
+
+## Enable and configure Azure Monitor
+
+Azure Monitor can collect data directly from your Arc-enabled servers into a Log Analytics workspace for detailed analysis and correlation. It requires installing the Azure Monitor Agent (AMA) VM extension in your Azure Arc-enabled servers, enabling VM insights to collect data from your machines.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:24]:
```suggestion
Azure Monitor can collect data directly from your Arc-enabled servers into a Log Analytics workspace for detailed analysis and correlation. It requires installing the Azure Monitor Agent (AMA) VM extension on your Azure Arc-enabled servers, enabling VM insights to collect data from your machines.
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 7]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+    ![Screenshot of searching for an Azure Arc-enabled server](./img/search_arc_server.png)
+
+- Click on **Insights** and then on **Performance**. You will find a set of performance charts that target several key performance indicators to help you determine how well your Azure Arc-enabled server is performing. The charts show resource utilization over a period of time:
+
+    ![Screenshot of VM insights - Performance](./img/vminsights_performance.png)
+
+## Operating System (OS) Performance Workbook
+
+An Azure Monitor Workbook for Operating System (OS) Performance is also available as part of the deployment. It complements the views provided by VM insights.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:64]:
```suggestion
An Azure Workbook for Operating System (OS) Performance is also available as part of the deployment. It complements the views provided by VM insights.
```
I think they are called Azure Workbooks now

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 8]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+  ![Screenshot of searching for clicking on Inventory Workbook](./img/click_inventoryworkbook.png)
+
+  ![Screenshot of opening Inventory workbook](./img/open_inventoryworkbook.png)
+
+- You will find the first inventory for **Machines overall status & configurations**, which counts the number of machines by type and by their status:
+
+  ![Screenshot of inventory workbook machines overall status & configurations](./img/machinesoverall_inventoryworkbook.png)
+
+- The second and third inventories list your **Azure Arc-enabled servers** and your **Azure Arc-enabled Kubernetes clusters**. It provides information about the status, the Arc agent version, the Operating System; location; and the number of compliant and non-compliant policies:
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md:102]:
```suggestion
- The second and third inventories list your **Azure Arc-enabled servers** and your **Azure Arc-enabled Kubernetes clusters**. It provides information about the status, Arc agent version, the operating system, location, and the number of compliant and non-compliant policies.
```

**Review by sebassem — CHANGES_REQUESTED** [review]
LGTM
@alsanch 
Very minor changes

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md [hunk 9]
```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
...
+
+  ![Screenshot of inventory workbook arc servers and kubernetes clusters](./img/arcserverskubernetesclusters_inventoryworkbook.png)
+
+- There is also an inventory for checking the **Updates Data** of the servers:
+
+  ![Screenshot of inventory workbook updates data](./img/updatesdata_inventoryworkbook.png)
+
+- The last inventory will list any **Defender for Cloud active alerts**:
+
+  ![Screenshot of inventory workbook arc servers and kubernetes clusters](./img/defenderalerts_inventoryworkbook.png)
```
[the remark this answers is not in the document]
Comment (alsanch) [reply docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md]: Completed
image

**Pull request merged**